incident.go

```diff
@@ -653,4 +653,57 @@ func (c *Client) ManageIncidentAlertsWithContext(ctx context.Context, incidentID
 	return &result, nil
 }
 
-/* TODO: Create Status Updates */
+// IncidentStatusUpdate is a status update for the specified incident.
+type IncidentStatusUpdate struct {
+	ID        string    `json:"id,omitempty"`
+	Message   string    `json:"message,omitempty"`
+	CreatedAt string    `json:"created_at,omitempty"`
+	Sender    APIObject `json:"sender,omitempty"`
```

Comment:
I apparently forgot to add this comment to my review before submitting it...

Since we don't use this type to serialize JSON, and only to deserialize it, I don't think the `omitempty` tags are needed on this type.

Reply:
I removed omitempty tag. Thanks.